tests/README.md

```diff
@@ -0,0 +1,75 @@
+### Integrated tools for synthetic data generation
+
+PyTrousse aids developers in automated testing by creating simulated raw data in a controlled way along with the data expected after appropriate data wrangling operations.
+Particularly this tool can support the data transformation operator testing by inverting their behavior through error injection.
+
+Particularly raw and expected data are wrapped in a `TestDataset` object that, following PyTrousse behavior, can be modified by applying the inverted data transformation operators.
+```python
+>>> import pandas as pd
+from .datasim import from_pandas
+from .datasim_util import (
+    Compose,
+    InsertOutOfScaleValues,
+    ReplaceSubstringsByValue,
+    SubstringReplaceMapByValue,
+)
+
+>>> df = pd.DataFrame(
+    {"float_column": list(range(10)), "int_column": [0.2 * i for i in range(10)]}
+)
+test_dataset = from_pandas(df)
+insert_errors = Compose(
+    [
+        ReplaceSubstringsByValue(
+            column_names=["float_column"],
+            error_count=2,
+            replacement_map_list=[
+                SubstringReplaceMapByValue(
+                    substr_to_replace=".",
+                    new_substring=",",
+                    value_after_fix=".",
+                    fix_with_substring=True,
+                )
+            ],
+        ),
+        InsertOutOfScaleValues(
+            column_names=["int_column"],
+            error_count=2,
+            upperbound_increase=0.02,
+            lowerbound_increase=0.02,
```

Comment:
I guess it should be "decrease" since we are talking about lower bounds... but I'll see that in the PR which is introducing these changes